src/java.base/share/classes/sun/security/ssl/SSLConfiguration.java

```diff
@@ -108,9 +108,14 @@ final class SSLConfiguration implements Cloneable {
     static final int maxHandshakeMessageSize = GetIntegerAction.privilegedGetProperty(
             "jdk.tls.maxHandshakeMessageSize", 32768);
 
-    // Set the max certificate chain length to 10
-    static final int maxCertificateChainLength = GetIntegerAction.privilegedGetProperty(
-            "jdk.tls.maxCertificateChainLength", 10);
+    // Maximum allowed certificate chain length
+    static final int maxCertificateChainLength;
```

Comment:
I think this can just be a local variable now inside the static block.

Reply:
Good catch. Done.